helpers/all.js

```diff
@@ -12,22 +10,16 @@ const factory = () => {
     return function(...args) {
         // Take the last arg (which is a Handlebars options object) out of args array
         const opts = args.pop();
-
+        let result;
         // Check if all the arguments are valid / truthy
-        const result = _.all(args, function (arg) {
-            if (_.isArray(arg)) {
-                return !!arg.length;
-            }
-            // If an empty object is passed, arg is false
-            else if (_.isEmpty(arg) && _.isObject(arg)) {
-                return false;
+        for (let i = 0; i < args.length; i++) {
+            if (!common.truthCheck(args[i])) {
+                result = false;
+                break;
+            } else if (i === args.length - 1) {
```

Comment:
A more readable way to do this would be:
```
let result = true;
for (let i = 0; i < args.length; i++) {
  result = result && common.isTruthy(args[i]);
  if (!result) {
    break;
  }
}
```

Reply:
done.